modules/toast/react/stories/stories.tsx

```diff
@@ -81,4 +86,17 @@ storiesOf('Components|Popups/Toast/React', module)
     <div className="story">
       <ToastWrapper state={'Error'} />
     </div>
+  ))
+  .add('With close button', () => (
+    <div className="story">
+      <Toast onClose={() => console.warn('test')}>Your workbook was successfully processed.</Toast>
```

Comment:
just checking if you wanted to leave these `console.warn()`s here. You could also use [Storybook's action addons](https://github.com/storybookjs/storybook/tree/master/addons/actions).